src/Jackett.Common/Definitions/hdtime.yml

```diff
@@ -84,12 +94,12 @@ search:
     - path: torrents.php
   inputs:
     $raw: "{{ range .Categories }}cat{{.}}=1&{{end}}"
-    search: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ end }}{{ if .Query.DoubanID }}/{{ .Query.DoubanID }}{{ else }}{{ end }}{{ if or .Query.IMDBID .Query.DoubanID }}{{ else }}{{ .Keywords }}{{ end }}"
+    search: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ end }}{{ if or .Query.IMDBID .Query.DoubanID }} {{ else }}{{ .Keywords }}{{ end }}{{ if .Query.DoubanID }}{{ .Query.DoubanID }}{{ else }}{{ end }}"
```

Comment:
What is purpose of the space after `{{ if or .Query.IMDBID .Query.DoubanID }}`? I think this would be trimmed anyway.

Unfortunately, I can't find the original discussion around the inclusion of a preceding `/`, but essentially it was deemed a good idea when searching the description to prevent potential matches to some other unrelated series of numbers (akin to IMDB's `tt`).

Reply:
Oh my bad, mind blank on my part, I was completely forgetting that they could be searched for at the same time. Thanks.